composer.json

```diff
@@ -14,7 +14,7 @@
     "require": {
         "ext-json": "*",
         "php": ">=8.0",
-        "gacela-project/gacela": "^0.19",
+        "gacela-project/gacela": "dev-master",
```

Comment:
We need to wait for `gacela@0.21`. Tomorrow or early next week will be out, so we can ping the version in this PR as well. @JesusValera can take care of this.